lib/inc/RedisChannel.h

```diff
@@ -17,49 +18,42 @@
 
 namespace sairedis
 {
-    class RedisChannel
+    class RedisChannel:
+        public Channel
     {
-        public:
-
-            typedef std::function<void(const std::string&,const std::string&, const std::vector<swss::FieldValueTuple>&)> Callback;
-
         public:
 
             RedisChannel(
                     _In_ const std::string& dbAsic,
-                    _In_ Callback callback);
+                    _In_ Channel::Callback callback);
 
             virtual ~RedisChannel();
 
         public:
 
             std::shared_ptr<swss::DBConnector> getDbConnector() const;
 
-            void setBuffered(
-                    _In_ bool buffered);
+            virtual void setBuffered(
```

Comment:
>virtual [](start = 12, length = 7)

remove `virtual` and use `override` only in derived class.

Reply:
not following?

Reply:
ref: https://stackoverflow.com/a/39932616

Reply:
i will still keep virtual and override keywords in child class, its consistent with all there is right now,
also for example in your case, if in child class function is marked as virtual foo() override, and base class foo() prototype changes, compiler generate error that there is no function to override from base class, so no mistake here if suddenly foo will be different function in child classed